test/unit/models/tasks.test.js

```diff
@@ -70,6 +71,63 @@ describe("tasks", function () {
       }
     });
   });
+
+  describe("fetchTasks", function () {
+    beforeEach(async function () {
+      const tasksPromise = tasksData.map(async (task) => {
+        await tasks.updateTask(task);
+      });
```

Comment:
Nit: Hi @bharati-21 want to ask what updateTask function is doing??
I seen the code but didn't understand

Reply:
It takes reference to a task. If it exists updates the task with it or creates a new task. Here using it to create a new task as we want some mock data.